### DISCUSSION TITLE
Add the ability change theme based on system settings (macOS)

### DISCUSSION THREAD

**Discussion by jeffreyguenther** [wezterm/wezterm#2878] [Ideas]
On macOS it would be lovely, to have the ability to set a dark and light theme so as the OS UI changes, terminal colors will be updated.

I'm not a Linux or Windows user so I don't know if such a thing exists there, but it would be a nice bit of polish.

**Comment by wez** [comment]
This is already supported on all platforms. See https://wezfurlong.org/wezterm/config/lua/wezterm.gui/get_appearance.html

**Comment by jeffreyguenther** [comment]
Oh, wonderful! Seems my search-foo failed me.

Thank you for such an awesome tool! Really digging it!
